babel.config.js

```diff
@@ -1,3 +1,17 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
+    function ModifyPackageJsonImportForTranspilation() {
+      return {
+        visitor: {
+          ImportDeclaration(path) {
+            let importResource = path.node.source.value ?? '';
+            if (importResource.includes('../package.json')) {
+              importResource = '../' + importResource;
```

Comment:
hm actually i think this isn't assign the new path to the actual `path.node.source.value`, will commit another fix